### PR TITLE
Remove virustotal feature and dashboards in favor of malware dashboard

### PR DESCRIPTION
**Pull request by guidomodarelli**
### Description

This pull request removes the `VirusTotal` application in favor to `Malware Detection` that are included there and move the VirusTotal sample data to Malware Detection category and adds YARA to the description.
 
### Issues Resolved
https://github.com/wazuh/wazuh-dashboard-plugins/issues/7027

#### Before
Screenshot_20240930_110640
image

#### After
image
image

### Test
Navigate to the overview page and ensure that the VirusTotal feature and dashboards do not exist.

### Check List
- [x] All tests pass
  - [x] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by Machi3mfl — APPROVED** [review]
CR: ✅ 
Test: ✅ 

### Before

Screenshot 2024-09-30 at 2 15 58 PM


### After

Screenshot 2024-09-30 at 2 15 43 PM

**Review by JuanGarriuz — DISMISSED** [review]
## Test 🟢 

image

File: plugins/main/public/utils/applications.ts
Comment (JuanGarriuz) [inline plugins/main/public/utils/applications.ts:1]: If we don't remove the constant from here, at least comment on it to explain that it is no longer used.

- line 235
`export const virustotal = {`

**Review by JuanGarriuz — DISMISSED** [review]
LGTM!!

**Review by JuanGarriuz — DISMISSED** [review]
LGTM!!

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ All notable changes to the Wazuh app project will be documented in this file.
 
 - Removed agent RBAC filters from dashboard queries [#6945](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6945)
 - Removed GET /elastic/statistics API endpoint [#7001](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7001)
+- Removed virustotal feature and dashboards in favor of malware dashboard [#7038](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7038)
```
Comment (Desvelao) [inline CHANGELOG.md:38]:
```suggestion
- Removed VirusTotal application in favor of Malware Detection [#7038](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7038)
```

File: plugins/main/README.md [hunk 2]
```diff
@@ -26,7 +26,6 @@ the Wazuh Indexer. The plugin provides the following capabilities:
   - Threat Detection and Response
     - Vulnerabilities: Discover what applications in your environment are affected by well-known vulnerabilities.
     - MITRE ATT&CK: Explore security alerts mapped to adversary tactics and techniques for better threat understanding.
-    - VirusTotal: Alerts resulting from VirusTotal analysis of suspicious files via an integration with their API.
```
Comment (Desvelao) [inline plugins/main/README.md:29]: **nitpick**: the Malware Detection application can display data about VirusTotal, so this should not be removed

File: plugins/main/public/controllers/management/components/management/configuration/configuration-settings.js [hunk 3]
```diff
@@ -80,13 +80,6 @@ export default [
         goto: 'alerts',
         when: 'manager',
       },
-      {
-        name: 'Integrations',
-        description:
-          'Slack, VirusTotal and PagerDuty integrations with external APIs',
-        goto: 'integrations',
-        when: 'manager',
-      },
```
Comment (Desvelao) [inline plugins/main/public/controllers/management/components/management/configuration/configuration-settings.js:89]: **nitpick**: this is related to the configuration of the Wazuh server and this should not be modified.

File: plugins/main/public/controllers/management/components/management/configuration/integrations/integrations.js [hunk 4]
```diff
@@ -22,12 +22,6 @@ const helpLinks = [
     text: 'Integration with external APIs',
     href: webDocumentationLink('user-manual/manager/manual-integration.html'),
   },
-  {
-    text: 'VirusTotal integration',
-    href: webDocumentationLink(
-      'user-manual/capabilities/malware-detection/virus-total-integration.html',
-    ),
-  },
```
Comment (Desvelao) [inline plugins/main/public/controllers/management/components/management/configuration/integrations/integrations.js:30]: **nitpick**: this is related to the configuration of the Wazuh server and this should not be modified.

File: plugins/main/public/controllers/management/components/management/configuration/integrations/integrations.js [hunk 5]
```diff
@@ -38,10 +32,6 @@ const helpLinks = [
 
 const defaultIntegrations = [
   { title: 'Slack', description: 'Get alerts directly on Slack' },
-  {
-    title: 'VirusTotal',
-    description: 'Get notified when malicious software is found',
-  },
```
Comment (Desvelao) [inline plugins/main/public/controllers/management/components/management/configuration/integrations/integrations.js:44]: **nitpick**: this is related to the configuration of the Wazuh server and this should not be modified.

File: plugins/main/public/controllers/management/components/management/configuration/integrations/integrations.js [hunk 6]
```diff
@@ -94,28 +84,26 @@ class WzConfigurationIntegrations extends Component {
...
-                      items={integrationsSettings}
-                      config={integration}
-                      help={key === 0 ? helpLinks : undefined}
-                    />
-                  </Fragment>
-                );
-              })
-          )
-          : null
-        }
```
Comment (Desvelao) [inline plugins/main/public/controllers/management/components/management/configuration/integrations/integrations.js:118]: **nitpick**: this is related to the configuration of the Wazuh server and this should not be modified.

File: scripts/wazuh-alerts-generator/cli.js [hunk 7]
```diff
@@ -108,13 +108,7 @@
...
     ],
     'auditing-policy-monitoring': ['audit', 'ciscat', 'openscap', 'rootcheck'],
-    'thread-detection': [
-      'docker',
-      'mitre',
-      'osquery',
-      'virustotal',
-      'vulnerabilities',
-    ],
+    'thread-detection': ['docker', 'mitre', 'osquery', 'vulnerabilities'],
```
Comment (Desvelao) [inline scripts/wazuh-alerts-generator/cli.js:111]: **nitpick**: I guess the sample data generator should be able to generate alerts related to VirusTotal.

File: plugins/main/public/components/add-modules-data/sample-data.tsx [hunk 8]
```diff
@@ -51,7 +50,6 @@ const sampleSecurityInformationApplication = [
 
 const sampleThreatDetectionApplication = [
   vulnerabilityDetection.title,
-  virustotal.title,
```
Comment (Desvelao) [inline plugins/main/public/components/add-modules-data/sample-data.tsx:54]: **nitpick**: the VirusTotal data should be visible through the Malware detection dashboard, so the sample data could be useful.

File: plugins/main/public/utils/applications.ts [hunk 9]
```diff
@@ -232,31 +232,6 @@ export const mitreAttack = {
     }`,
 };
 
-export const virustotal = {
```
Comment (Desvelao) [inline plugins/main/public/utils/applications.ts:235]: **nitpick**: I think this variable is used (title at least) when you restored this: https://github.com/wazuh/wazuh-dashboard-plugins/pull/7038/commits/52565172b44565c5d4baf045c642d17557c968c4.

You could remove the variable and add the string in the expected view if we consider or remove it.

**Review by Desvelao — APPROVED** [review]
LGTM!

**Pull request merged**